src/Core/src/Handlers/MenuFlyoutItem/MenuFlyoutItemHandler.cs

```diff
@@ -24,6 +24,8 @@ public partial class MenuFlyoutItemHandler : ElementHandler<IMenuFlyoutItem, Pla
 			[nameof(IMenuFlyoutSubItem.Text)] = MapText,
 			[nameof(IMenuElement.Accelerators)] = MapAccelerator,
 			[nameof(IMenuElement.Source)] = MapSource,
+#endif
+#if MACCATALYST || WINDOWS
```

Comment:
Just confirming that this is **NOT** neeeded/used on iOS? I see the `MenuBarItemHandler` has a mapper for iOS, and this function appears to compile for iOS too?